package.json

```diff
@@ -166,6 +168,7 @@
     "eslint-rule-composer": "^0.3.0",
     "eslump": "^3.0.0",
     "esprima": "^4.0.1",
+    "execa": "^9.5.2",
```

Comment:
[Question] I saw the existing usage of `cross-spawn`, but personally find the execa usage in `test-ecosystem.mjs` to be ergonomic (it's also what I'm used to). I think it should be not too much work to switch back over. Does the team have a stance on the current preferred way to run shell scripts?

Reply:
I think it's fine to stick with `execa`.

Reply:
[nano-spawn](https://github.com/sindresorhus/nano-spawn)? Same author.

Reply:
I like it 🙂